### PR TITLE
Replace rvv-as

### PR DESCRIPTION
**Pull request by TheWaWaR**

### CODE REVIEW

File: make.py [hunk 1]
```diff
@@ -108,40 +108,40 @@
 
 assert len(glob.glob('res/*.c')) == len(c_entry)
 
+def run(entry):
+    print(f'run bin/{entry}')
+    s = subprocess.call(f"ckb-debugger-rvv --bin bin/{entry} | fgrep 'Run result: 0'", shell=True)
```
Comment (mohanson) [inline make.py]: Don't use debugger here, the goal of rvv-playground is to execute test cases during development, using debugger will make it all very complicated.

Comment (TheWaWaR) [reply make.py:113]: ok

Comment (TheWaWaR) [reply make.py]: Can I use `rvv` branch of `ckb-vm/examples/int64` ?

Comment (mohanson) [reply make.py]: yes, keep it as is

**Review by mohanson — APPROVED** [review]

**Pull request merged**